README.md

```diff
@@ -1,33 +1,45 @@
-If you want to perform a new DE analysis in a different biological **context** (i.e. different organism, tissue, media) then you might not have the curated data available. Switching contexts will require a lot of manual effort. Similarly, using a different statistical method will require re-curation effort
+If you want to perform a new DE analysis in a different biological **context** (i.e. different organism, tissue, media) then you might not have the curated data available. Switching contexts will require re-curation. Similarly, using a different statistical method will require re-curation. This curation effor it very time intensive.
 
 
-**Goal of this study:**
-* To show that our compendia simulation method, [ponyo](https://github.com/greenelab/ponyo) can automatically identify generic genes and pathways
+**Goal:**
+To develop a method that can automatically identify generic genes and pathways
 
 **Results:**
-* We found a set of general generic genes (i.e. genes found to be generic in both recount2 and crow et. al., which contain a mix of experiments)
-* We developed a method to automatically identify generic genes in different contexts without having to perform experiments and curate. 
+Our method ranking was consistent with previously published ranking. These generic genes appear to act as gene hubs, which are associated with many biological processes. 
+
+**Conclusions:**
+We developed a method to automatically identify generic genes and pathways using public data without the need for curation. The generic signals identified from this method can be used to interpret study results and direct follow-up experiments.
+
+## Directory Structure
+| Folder/file | Description |
+| --- | --- | 
+| [configs](configs) | This folder contains configuration files used to set hyperparameters for the different experiments |
+| [generic_expression_patterns_modules](generic_expression_patterns_modules) | This folder contains supporting functions that other notebooks in this repository will use |
+| [human_cancer_analysis](human_cancer_analysis) | This folder contains analysis notebooks to identify generic signals using Powers et. al. dataset to train VAE |
+| [human_general_analysis](human_general_analysis) | This folder contains analysis notebooks to identify generic signalsusing [recount2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6742427/) dataset to train VAE |
+| [multiplier_analysis](multiplier_analysis) | This folder contains analysis notebooks to coverage of generic genes across [MultiPLIER latent variables](https://www.cell.com/cell-systems/pdfExtended/S2405-4712\(19\)30119-X)) |
+| [pseudomonas_analysis](pseudomonas_analysis) |  This folder contains analysis notebooks to identify generic signalsusing *P. aeruginosa* dataset to train VAE |
+
 
+## Usage
 
-## How to run notebooks from generic-expression-patterns
+**How to run notebooks from generic-expression-patterns**
 
-**Operating Systems:** Mac OS, Linux
+*Operating Systems:* Mac OS, Linux
```

Comment:
Is this just because you haven't tested on Windows? Might be good to just add a note "(not tested on Windows)" so people don't assume there's some known reason it won't work (unless there is and I'm missing it).

Reply:
Good question. I did try to test this on windows and there were some conda libraries that weren't available. I'll make a note of that.